Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+#!groovy
+
+pipeline {
+    agent none
+
+    stages {
+        stage('clean') {
+            agent { label 'master' }
+            steps {
+                sh 'git clean -fdx'
+            }
+        }
+
+        stage('container') {
+            agent {
+                dockerfile {
+                    args '-v ${HOME}/.eggs:${WORKSPACE}/.eggs'
+                }
+            }
+            environment {
+                HOME = '.'
+            }
+            stages {
+                stage('version') {
+                    steps {
+                        sh 'bumpversion patch'
+                    }
+                }
+                stage('release') {
+                    when { branch 'master' }
+                    steps {
+                        sh 'bumpversion --tag --commit release'
```

Comment:
git will need to be installed in the docker to make this work

Reply:
Ah yeah, good point. We might need to refactor it to run outside the container, since it just operates against the git checkout, and isn't actually part of the build toolchain anyway.